File: miplanv.client/src/models/PackedLunch.ts
export interface BaseEntity
{
    id: number,
    createdAt: string,
    isActive: boolean
}

export interface PackedLunch extends BaseEntity {
    name: string;
    isVegan: boolean;
    image: string | null;
    description: string | null;
    isCurrent: boolean;
    orderDetails: any[];
}

export interface CurrentPackedLunch {
    id: number;
    name: string;
    description: string | null;
    image: string | null;
    isVegan: boolean;
}

File: MiPlanV.Domain/Entities/PackedLunch.cs
using MiPlanV.Domain.Common;

namespace MiPlanV.Domain.Entities;

public class PackedLunch : BaseEntity
{
    public string Name { get; set; } = string.Empty;
    public bool IsVegan { get; set; }
    public string? Image { get; set; }
    public string? Description { get; set; } = string.Empty;
    public bool IsCurrent { get; set; }
    public virtual ICollection<OrderDetail> OrderDetails { get; set; }
}
